src/components/Reports/StaffExpenseReport/DownloadButtonGroup/downloadReport.tsx

```diff
@@ -0,0 +1,68 @@
+import { buildURI } from 'react-csv/lib/core';
+import { Transaction } from 'src/components/Reports/StaffExpenseReport/StaffExpenseReport';
+import { ReportType } from '../Helpers/StaffReportEnum';
+
+const makeTable = (
+  title: string,
+  transactions: Transaction[],
+  // TFunction to translate text
+  t: (key: string) => string,
+) => {
+  const csvHeader = [t('Date'), t('Category'), t('Amount')];
+
+  const csvData = [
+    [title],
+    csvHeader,
+    ...transactions.map((transaction) => [
+      transaction.month,
+      transaction.category,
+      transaction.total.toString(),
+    ]),
+  ];
+
+  return csvData;
+};
+
+export const downloadCsv = (
+  type: ReportType,
+  enqueueSnackbar: (message: string, options?: { variant: string }) => void,
+  transactions: Transaction[],
+  // translation function
+  t: (key: string) => string,
```

Comment:
You can import the type from `react-i18next`.

```suggestion
  t: TFunction,
```